packages/edge/README.md

```diff
@@ -0,0 +1,55 @@
+
+# React Native AEP Edge Network
+
+[![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aepedge.svg)](https://www.npmjs.com/package/@adobe/react-native-aepedge) 
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepedge)](https://www.npmjs.com/package/@adobe/react-native-aepedge)
+
+## Usage
+
+### [Edge Network](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension)
+
+#### Importing the extension:
+```javascript
+import {AEPEdge} from '@adobe/react-native-aepedge';
```

Comment:
This should be `import {AEPEdge, AEPExperienceEvent} from '@adobe/react-native-aepedge';`